src/ZPublisher/tests/test_mapply.py

```diff
@@ -51,13 +54,15 @@ def __call__(self, b, c=4):
         v = mapply(cc.compute, (), values)
         self.assertEqual(v, '334')
 
-        class c2:
```

Comment:
Couldn't this be split into a separate testcase, using `unittest.skipUnless(PY2)`.